README.markdown

```diff
@@ -1,29 +1,28 @@
-stylish-haskell
-===============
+## stylish-haskell
+
+<img src="./assets/Logo/SVG/RoundedLogo.svg" width="100px">
```

Comment:
Very cool logo indeed 👌 hope that @jaspervdj likes it! ❤️ 

![stylish-haskell](https://user-images.githubusercontent.com/5127501/69405658-87ef7600-0d00-11ea-8b40-93aebd4d752a.png)

Reply:
Thank you Flavio!!